zfit/zfit
# Add errors computed from loss function

Fixes #181 


## Proposed Changes

  - computation of asymmetric errors from the profiling of the loss function, working with any minimiser
  - this would replace `minos` as the basic function to compute the errors
  - can find new minimum while scanning the loss function, in that case the current fit result attribute `valid` is set to false. The new fit result is returned.

## Tests added

  - check that errors from `minos` and the new function are close/similar
  - check cases when a new minimum is found
  -
  
## Checklist

 - [x] change approved
 - [x] implementation finished
 - [x] correct namespace imported
